lib/ngtsc/workspace.symbols.ts

```diff
@@ -0,0 +1,476 @@
+        this.injectableRegistry,
+        this.isCore,
+        !!this.options.annotateForClosureCompiler
+      )
+    );
+  }
+
+  /** Handler for @NgModule() annotations */
+  get moduleHandler() {
+    return this.lazy('moduleHandler', () => new NgModuleDecoratorHandler(
+        this.reflector,
+        this.evaluator,
+        this.metaReader,
+        this.metaRegistry,
+        this.scopeRegistry,
+        this.referencesRegistry,
+        this.isCore,
+        this.routeAnalyzer,
+        this.refEmitter,
+        this.host.factoryTracker,
+        this.defaultImportTracker,
+        !!this.options.annotateForClosureCompiler,
+        this.injectableRegistry,
+        this.options.i18nInLocale
+      )
+    );
+  }
+
+  /** Handler for @Component() annotations */
+  get cmptHandler() {
+    return this.lazy('cmptHandler', () => new ComponentDecoratorHandler(
+        this.reflector,
+        this.evaluator,
+        this.metaRegistry,
+        this.metaReader,
+        this.scopeReader,
+        this.scopeRegistry,
+        this.isCore,
+        this.resourceManager,
+        this.host.rootDirs,
+        this.options.preserveWhitespaces || false,
+        this.options.i18nUseExternalIds !== false,
+        this.options.enableI18nLegacyMessageIdFormat !== false,
+        this.moduleResolver,
+        this.cycleAnalyzer,
+        this.refEmitter,
+        this.defaultImportTracker,
+        this.incrementalDriver.depGraph,
+        this.injectableRegistry,
+        !!this.options.annotateForClosureCompiler,
+      )
+    );
+  }
+
+  /** Static reflection of declarations using the TypeScript type checker */
+  public get reflector() {
+    return this.lazy('reflector', () => new TypeScriptReflectionHost(this.checker));
+  }
+
+  /** Evaluate typecript Expression & update the dependancy graph accordingly */
+  public get evaluator() {
+    return this.lazy('evaluator', () => new PartialEvaluator(
+      this.reflector,
+      this.checker,
+      this.incrementalDriver.depGraph
+    ));
+  }
+
+  /** Register metadata from local NgModules, Directives, Components, and Pipes */
+  public get metaRegistry() {
+    return this.lazy('metaRegistry', () => new CompoundMetadataRegistry([ this.localMetaReader, this.scopeRegistry ]));
+  }
+
+  /** Register metadata from local declaration files (.d.ts) */
+  public get metaReader() {
+    return this.lazy('metaReader', () => new CompoundMetadataReader([ this.localMetaReader, this.dtsReader ]));
+  }
+
+  /** Collects information about local NgModules, Directives, Components, and Pipes (declare in the ts.Program) */
+  public get scopeRegistry() {
+    return this.lazy('scopeRegistry', () => {
+      const depScopeReader = new MetadataDtsModuleScopeResolver(this.dtsReader, this.aliasingHost);
+      return new LocalModuleScopeRegistry(this.localMetaReader, depScopeReader, this.refEmitter, this.aliasingHost);
+    });
+  }
+
+  /** Analyze lazy loaded routes */
+  public get routeAnalyzer() {
```

Comment:
We can remove this for now. The Guess.js parser does this, so we don't have to couple to unstable APIs.